src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/CallInstruction.Generated.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Linq.Expressions.Interpreter
     internal partial class CallInstruction
```

Comment:
>CallInstruction [](start = 27, length = 15)

It is not clear to me how this file is "generated". Is it just an inaccurate name?

Reply:
It looks like it was generated, but I do not know by what. We switched some other generated files to be produced via T4 templates. 
I think it would be a good idea for this one as well, eventually, but not in this change.

---
In reply to: [181904387](https://github.com/dotnet/corefx/pull/28792#discussion_r181904387) [](ancestors = 181904387)